src/server/server-middlewares.js

```diff
@@ -3,23 +3,43 @@ import { Router } from 'express'
 import type { $Request, $Response, NextFunction } from 'express'
 import bodyParser from "body-parser"
 import cors from "cors"
+import pino from 'express-pino-logger'
 import addLoginMiddlewares from "./login/login-middleware"
-import { setup as addGunMiddlewares } from "./gun/gun-middleware"
+import { setup as addGunMiddlewares, GunDBPrivate } from "./gun/gun-middleware"
 import addStorageMiddlewares from "./storage/storageAPI"
 import addVerificationMiddlewares from "./verification/verificationAPI"
+import VerificationAPI from "./verification/verifications"
 
-import { GunDBPrivate } from "./gun/gun-middleware"
-import logger from "../imports/pino-logger"
+import logger from '../imports/pino-logger'
+import conf from './server.config'
 
 function wrapAsync(fn: Function) {
-  return function (req: $Request, res: $Response, next: NextFunction) {
+  return function (req: $Request & { log: any }, res: $Response, next: NextFunction) {
+    const log = req.log.child({ from: 'wrapAsync' })
```

Comment:
Here maybe its ok to use the express logger, to show each request details.
But in general inside the handlers themselves the logger from the "req" is too much information, lets use the regular pino logger inside the handlers

Reply:
Oh, sure... temp logging committed, sorry.

It was useful for me to log the errors as, sometimes, I got a simple `ERROR {}` logged and this helped me a lot to easily find them and committed.

Reply:
i'm not referring to the temp logging, the pino-express extension adds a logger to the "req" object. this logger logs lotsa info about the http request which we don't need for every log.debug, so its better to use a per file logger "import logger from './logger/pino-logger'"

Reply:
Oh, if that's the issue we can just update the pino options and remove `req`, `res` from logging.

![image](https://user-images.githubusercontent.com/3315606/52709477-96d41b80-2f6b-11e9-96a2-ba247aa8f106.png)